kotlinx-coroutines-core/common/src/Debug.common.kt

```diff
@@ -27,7 +27,7 @@ internal expect fun assert(value: () -> Boolean)
  * Copy mechanism is used only on JVM, but it might be convenient to implement it in common exceptions,
  * so on JVM their stacktraces will be properly recovered.
  */
-@ExperimentalCoroutinesApi
+@ExperimentalCoroutinesApi // Since 1.2.0, no ETA on stability
```

Comment:
What is needed for `CopyableThrowable` to become stable? Is there any issue to watch? Can we help somehow?

This came to my attention, because Retrofit now produces not very helpful stacktraces from suspending calls and the corresponding fix is blocked because this is experimental: https://github.com/square/retrofit/issues/3474

Reply:
The best way is to let us know and file an issue. Filed https://github.com/Kotlin/kotlinx.coroutines/issues/2367